kustomize/commands/openapi/fetch/fetch.go

```diff
@@ -46,5 +59,14 @@ Installation and setup instructions: https://kubernetes.io/docs/tasks/tools/inst
 	output := stdout.Bytes()
 	json.Unmarshal(output, &jsonSchema)
 	output, _ = json.MarshalIndent(jsonSchema, "", "  ")
+
+	if format == "yaml" {
+		output, err = yaml.JSONToYAML(output)
+		if err != nil {
+			fmt.Fprintln(w, err)
```

Comment:
Same question here: since this is an error, can we output to stderr?

Reply:
done